### DISCUSSION TITLE
Add global option to disallow Update of beatmaps to change date added

### DISCUSSION THREAD

**Discussion by EXtremeExploit** [ppy/osu#19731] [Ideas] [answered]
MANY users sort by date added in osu stable, whenever a beatmap has an update its common to note the date the map was modified, update and change the date back to how it was to maintain the order in song selection, it is really annoying (both on lazer and stable) that everytime i update a beatmap to have to manually change the dates, with lazer this is impossible because of how it stores beatmaps, also this could fallback to implementing per beatmap date edited and not be stored as a beatmap set thing

**Comment by Kyorie** [comment]


**Reply by EXtremeExploit** [reply]
so maps store 2 dates? date added (which would be static and constant) and date updated which can vary?

**Comment by peppy** [comment, marked as answer]
"Date added" should not be updated when you update a Beatmap.


https://user-images.githubusercontent.com/191335/184467173-4ba719bc-a2c5-4051-89ec-0ece664feac9.mp4


Can you show what you're seeing and provide logs?

**Reply by EXtremeExploit** [reply]
ah my bad, i am not very good at looking
